addons/mk6mortar/functions/fnc_turretDisplayLoaded.sqf

```diff
@@ -45,13 +45,7 @@ private _fnc_hideControl = {
     if (_hideCtrl) then {
         _pos = [-9,-9,0,0];
     } else {
-        {
-            if (isNumber (_path >> _x)) then {
-                _pos pushBack (getNumber (_path >> _x));
-            } else {
-                _pos pushBack (call compile (getText (_path >> _x)));
-            };
-        } forEach ["x", "y", "w", "h"];
+        _pos = ["x", "y", "w", "h"] apply {getNumber (_path >> _x)};
```

Comment:
No longer supports globals, as they are read from parsing namespace, unless you change it to the current namespace via call compile.

Reply:
I have no clue why I wrote it like that, this change should be fine

Reply:
The config entries are for Arma Engine. And Arma reads it with parsingNamespace too.  And it works for Arma too. So no problem here.

Reply:
You're right. I was thinking about config entries read by the engine and not getNumber. OOps.